.circleci/config.yml

```diff
@@ -34,14 +35,13 @@ jobs:
           command: |
             ./tests/e2e.sh kind
             ./tests/e2e.sh deploy-operator
-            ./tests/e2e.sh test
+            ./tests/e2e.sh test --tls
```

Comment:
Do these still work without tls too? If so can we `test` and `test --tls`?

Reply:
yes, that's how the non-operator test works :))